### PR TITLE
docs: improvement to “Get Started” section

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

File: content/docs/en/get-started/configure-workspace/overview.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
 title: Configure the Workspace
 ---
 
-You've already deployed Bytebase to proudction, before making any changes to databases, you should first configure the workspace.
+Now that you have successfully deployed Bytebase. Before making any changes to databases, you should first configure the workspace.
 
-The entire Bytebase instance hosts a single **Workspace**. At the workspace level you can configure global settings applicable to all later created projects. There are three roles at the workspace level: **Owner**, **DBA**, and **Developer**.
+A single Bytebase deployment represents a single **Workspace**. At the workspace level you can configure global settings applicable to all later created projects. There are three roles at the workspace level: **Owner**, **DBA**, and **Developer**.
```
Comment (milasuperstar) [inline content/docs/en/get-started/configure-workspace/overview.md:7]:
```suggestion
A single Bytebase deployment represents a single **Workspace**. At the workspace level, you can configure global settings applicable to all projects within this workspace. There are three roles at the workspace level: **Owner**, **DBA**, and **Developer**.
```

File: content/docs/en/get-started/configure-workspace/register-accounts.md [hunk 2]
```diff
@@ -10,21 +10,26 @@ The first registered account after deployment will be granted a **Workspace Owne
 
 ![welcome-page](/docs/en/get-started/configure-workspace/register-accounts/welcome-page.webp)
 
-## **Register an account**
+## **Register a normal account**
```
Comment (milasuperstar) [inline content/docs/en/get-started/configure-workspace/register-accounts.md:13]:
```suggestion
## **Register a regular account**
```
normal sounds a bit weird 😂

**Review by milasuperstar — CHANGES_REQUESTED** [review]

File: content/docs/en/get-started/configure-workspace/register-accounts.md [hunk 3]
```diff
@@ -10,21 +10,26 @@ The first registered account after deployment will be granted a **Workspace Owne
 
 ![welcome-page](/docs/en/get-started/configure-workspace/register-accounts/welcome-page.webp)
 
-## **Register an account**
+## **Register a regular account**
 
 After the admin account is created, you can register a normal account and will be granted **Workspace Developer** role.
```
Comment (tianzhou) [inline content/docs/en/get-started/configure-workspace/register-accounts.md:15]:
```suggestion
After the admin account is created, you can register a regular account and will be granted **Workspace Developer** role.
```

File: content/docs/en/get-started/configure-workspace/register-accounts.md (hunk 2 above)
Comment (tianzhou) [reply content/docs/en/get-started/configure-workspace/register-accounts.md]: Indeed

**Review by tianzhou — COMMENTED** [review]
PTAL

**Review by milasuperstar — APPROVED** [review]

**Pull request merged**